packages/chart/src/Pie.ts

```diff
@@ -230,6 +235,10 @@ export class Pie extends SVGWidget {
         SVGWidget.prototype.exit.apply(this, arguments);
     }
 
+    getPieGenerator() {
```

Comment:
This confused me at first - can you:
* Change to "updateD3Pie(): void"
* move the following code into it https://github.com/hpcc-systems/Visualization/blob/4eb8e4698f57a3d0955e990fd44644a034eee189/packages/chart/src/Pie.ts#L31-L37 into it
* Then call it at the start of the update method

Then the other classes can override the method as needed.